src/main/webapp/resources/js/apis/projects/projects.js

```diff
@@ -12,17 +12,15 @@ export const projectsApi = createApi({
   baseQuery: fetchBaseQuery({ baseUrl: URL }),
   tagTypes: ["Projects"],
   endpoints: (build) => ({
-    getProjectsForUser: build.query({
-      query: (query) => ({
-        url: "/user",
-        params: { query },
+    getProjectsToShareTo: build.query({
```

Comment:
This should also be renamed in relation to the above comments

Reply:
Updated in 5630b02